Overview.bs

```diff
@@ -396,36 +428,49 @@ Offline Usage {#offline-usage}
-due to JavaScript execution). In these cases the page saving mechanism should download the full font
-by making a normal GET request without the [=patch request header=] to the font url. Additionally
-when URLs are rewritten to point to the saved full font any of the incremental tech specifiers should
-be removed.
+due to JavaScript execution). In these cases, when using patch-subset the page saving mechanism
+should download the full font by making a normal GET request without the [=patch request header=]
+to the font url. When using IFTB the page saving mechanism should download and integrate all
+remaining chunks.  Additionally, when URLs are rewritten to point to the saved full font any
+of the incremental tech specifiers should be removed.
 
-Patch Based Incremental Transfer {#patch-incxfer}
-=================================================
 
-Font Subset {#font-subset-info}
---------------------------
+Preliminary Definitions {#prelim-definitions}
+=============================================
 
-A <dfn dfn>font subset</dfn> is a modified version of a font file that contains only the data needed to
-render a subset of:
+Font Subset Description {#font-subset-des-def}
+----------------------------------------------
 
-*  the codepoints,
-*  <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags">OpenType layout
-    features</a>,
-*  and <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview#terminology">design-variation space</a>.
+A <dfn>font subset description</dfn> specifies the minimum data that a [=font subset=] must possess:
 
-supported by the original font.
-<span class="conform client server" id="conform-font-subset">
-When a subsetted font is used to render text using any combination of the subset codepoints,
-<a href="https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags">layout features</a>,
-or <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview#terminology">design-variation space</a> it must render identically to the original font. This includes rendering with
-the use of any optional typographic features that a renderer may choose to use from the original font,
-such as hinting instructions.
-</span>
+*  a set of codepoints,
+*  a set of <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags">opentype
+    layout features</a>,
+*  and (in the case of a variable font) a <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview#terminology">design-variation space</a>.
+
+Glyph Closure {#glyph-closure-def}
+----------------------------------
 
-A <dfn dfn>font subset definition</dfn> describes the minimum data (codepoints, layout features,
-variation axis space) that a [=font subset=] must possess.
+The <dfn>glyph closure</dfn> of a font subset description is the full set of glyphs in the
+font that may be needed to render any combination of the described codepoints and layout features.
```

Comment:
I believe it's also necessary to mentioned variation space here since variation space can influence glyph substitutions through things like rvrn.